dev-util/webstorm/webstorm-2019.1.191.6183.63.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils desktop
+
+MY_PN="WebStorm"
+MY_PV="$(ver_cut 1-2)"
+MY_BN="$(ver_cut 3-5)"
+
+DESCRIPTION="The powerful IDE for modern JavaScript development"
+HOMEPAGE="http://www.jetbrains.com/webstorm"
+SRC_URI="http://download.jetbrains.com/webstorm/${MY_PN}-${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="IDEA || ( IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="2019"
```

Comment:
Will this install parallel to previous slot without collisions? If not, use the previous slot.